.ci/test.sh

```diff
@@ -21,6 +21,7 @@ export PATH="$HOME/miniconda/bin:$PATH"
 
 ${CI_TOOLS}/lint_py.sh $(pwd)
 Rscript ${CI_TOOLS}/lint_r_code.R $(pwd)
+${CI_TOOLS}/lint_todo.sh $(pwd)
```

Comment:
I see a "Permission Denied" in the Travis logs. I think you need to `chmod +x .ci/lint_todo.sh`. There is a bit in each file that tracks whether or not it's executable like this.

Reply:
Man, speedy response.  

It sure does.  I added an inline TODO as well just in case. 

![image](https://user-images.githubusercontent.com/24531403/74599748-b903ec00-504c-11ea-9884-f193f35e876a.png)

Reply:
chmod done.